src/pages/coders.js

```diff
@@ -100,7 +100,13 @@ function Coder({ githubUser, avatar, approved, totalContributions }) {
       {showDetails && (
         <CoderDetails>
           <CoderText>
-            <CoderName>@{githubUser}</CoderName>
+            <CoderName>
+              <a 
+                href={`https://github.com/${githubUser}`}
+                target="_blank"
+                style={{text-decoration:none}}
```

Comment:
@Koalapvh13  Are you debugging the project locally?

I'm saying it because a simple `make start` after installing the dependencies (`make setup`)  shows:

``` 
107:29  error  Parsing error: Unexpected token, expected ","
```

And your commit was not formatted by Prettier.